src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/SizeOfTests.cs

```diff
@@ -59,6 +59,12 @@ public void SizeOf_Struct_With_GenericValueTypeField_ReturnsExpected()
             Assert.Equal(8, Marshal.SizeOf<TestStructWithVector64>());
         }
 
+        [Fact]
+        public void SizeOf_TypeWithEmptyBase_ReturnsExpected()
```

Comment:
Just to confirm, this test would have failed prior to the above change - correct?

Reply:
Yes, it would have failed since the SizeOf call would return 8.